app/tests/archives_tests/test_models.py

```diff
@@ -15,3 +16,40 @@ def create_archive_items_for_images(images, archive):
         civ = ComponentInterfaceValueFactory(image=image)
         ai = ArchiveItemFactory(archive=archive)
         ai.values.add(civ)
+
+
+@pytest.mark.django_db(transaction=True)
+def test_archive_item_set_title():
```

Comment:
This is the same for reader studies, right? Might be nice to combine the tests as well and parametrize them accordingly.

Reply:
Please do not add any transaction tests, they're hideously slow.

Reply:
I am a bit rusty with atomic interactions. These tests fail on the DB level (Integrity constraints) and the transaction kept hanging as a result. We could use a context manager, maybe (?), but I am not too sure if it's any quicker.

Reply:
Fixed with contextmanagering atomic transmissions. (Wow, that is a cool sentence.)